pkg/oci/utils_test.go

```diff
@@ -293,6 +293,50 @@ func TestStateToOCIState(t *testing.T) {
 	}
 }
 
+func TestEnvVars(t *testing.T) {
+	envVars := []string{"foo=bar", "TERM=xterm", "HOME=/home/foo"}
+	expectecVcEnvVars := []vc.EnvVar{
+		{
+			Var:   "foo",
+			Value: "bar",
+		},
+
+		{
+			Var:   "TERM",
+			Value: "xterm",
+		},
+
+		{
+			Var:   "HOME",
+			Value: "/home/foo",
+		},
+	}
+
+	vcEnvVars, err := EnvVars(envVars)
+	if err != nil {
+		t.Fatalf("Could not create environment variable slice %v", err)
+	}
+
+	if reflect.DeepEqual(vcEnvVars, expectecVcEnvVars) == false {
+		t.Fatalf("Got %v\n expecting %v", vcEnvVars, expectecVcEnvVars)
+	}
+}
+
+func TestMalformedEnvVars(t *testing.T) {
+	envVars := []string{"foo"}
+
+	_, err := EnvVars(envVars)
+	if err == nil {
+		t.Fatalf("Could not create environment variable slice %v", err)
```

Comment:
I think this message (and the one below) need to say instead something like "EnvVars() succeeded unexpectedly: ...".
Also, you are displaying `err` which is only ever going to be `nil` in that block.